airlock/types.py

```diff
@@ -12,3 +18,70 @@
     class UrlPath(PurePosixPath): ...
 else:
     UrlPath = PurePosixPath
+
+
+@dataclass
+class FileMetadata:
```

Comment:
Is types.py the right place for this?  What's the criteria for something going in this file?

Reply:
The criteria is that the type needed by both business_logic.py and file_browser_api.py. We can get in a mess with circular imports otherwise, especially since we're statically typing things.

It's basically the root of the type dependency tree.